deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1995,6 +2011,13 @@ def registry_key_cud(root_key, registry_sub_key, log_monitor, arch=KEY_WOW64_64K
         if triggers_event_add:
             logger.info("'added' {} detected as expected.\n".format("events" if len(key_list) > 1 else "event"))
 
+            # Update the position of the log to the end of the scan
+            if time_travel:
+                log_monitor.start(timeout=global_parameters.default_timeout, callback=callback_detect_end_scan,
+                                  update_position=True,
+                                  error_message=f'End of scheduled scan not detected after '
+                                  f'{global_parameters.default_timeout} seconds')
```

Comment:
```suggestion
                                  error_message=f"End of scheduled scan not detected after "
                                  f"{global_parameters.default_timeout} seconds")
```
When there is a variable inside a string, we usually use double quotes.

Reply:
Done 25b6b8e9a294aec722b3c8b4f042b11f05f9cc56